apps/pie-docs/src/_11ty/shortcodes/componentDetailsTable.js

```diff
@@ -59,15 +59,25 @@ const buildRow = (cells) => cells.map((cell) => {
  */
 module.exports = ({
     tableData,
+    priority,
 }) => {
     const { headings, rows } = JSON.parse(tableData);
+    const hasWidePadding = headings?.length <= 3;
 
     return `<div class="c-componentDetailsTable-backdrop">
-    <table class="c-componentDetailsTable">
-    ${headings ? `<tr>${headings.map((heading) => `
-        <th>${heading}</th>`).join('')}
+    <table class="c-componentDetailsTable ${hasWidePadding ? 'c-componentDetailsTable-hasWidePadding' : ''}">
+    ${headings ? `<tr>${headings.map((heading, index) => {
+        const isPriorityColumn = priority && priority === index;
```

Comment:
do we still need this change?
if not, we can clean it up in a follow up pr

Reply:
I'll do it on the next one, thanks :)